app/views/notifier/new_user.html.haml

```diff
@@ -2,8 +2,19 @@
 
 - if @password
   %p
+    - if text(".username_and_password_text").present?
+      = text(".username_and_password_text")
+      %br
```

Comment:
What would you think of something like this and rely on text_helper's markdown capabilities for the linebreaks/formatting?

```
- if @password
  = html("username_and_password_html" username: @user.username, password: @password)
- else
  = html("only_username_html", username: @user.username)
```